### ISSUE TITLE
Invalid grammar generated for concepts containing DateTime

### ISSUE THREAD

**Issue by dselman** [accordproject/cicero#406]
**Describe the bug**

Cannot generate a grammar for a template that references a concept containing a DateTime.

**To Reproduce**

Use studio.accordproject.org and add the following. In the JS console you see an error:
```
[Log] Error building grammar Error generating text from this new grammar (client.bundle.js, line 2185)
```

Model:
```
namespace org.accordproject.helloworld

import org.accordproject.cicero.contract.* from https://models.accordproject.org/cicero/contract.cto
import org.accordproject.cicero.runtime.* from https://models.accordproject.org/cicero/runtime.cto

transaction MyRequest extends Request {
  o String input
}

transaction MyResponse extends Response {
  o String output
}

concept MyConcept {
  o DateTime myDateTime
}

/**
 * The template model
 */
asset HelloWorldClause extends AccordClause {
  /**
   * The name for the clause
   */
  o String name
  o MyConcept myConcept
}
```

Template:
```
Name of the [{myConcept}] person to greet: [{name}].
Thank you!
```

Sample:

```
Name of the 10/10/2000 person to greet: "Fred Blogs".
Thank you!
```

**Expected behavior**
Grammar should work, and should parse the `DateTime` using the default MM/DD/YYYY format.

**Comment by dselman** [comment]
[helloworld-4.zip](https://github.com/accordproject/cicero/files/3534525/helloworld-4.zip)

**Comment by jeromesimeon** [comment]
This error still occurs on the `0.20` branch. Attached sample.
[helloworld-4.0.20.zip](https://github.com/accordproject/cicero/files/3725416/helloworld-4.0.20.zip)

**Comment by jeromesimeon** [comment]
Fixed in branch `0.20` https://github.com/accordproject/cicero/commit/e4d578f67f705c64ea7fedab204b9efd05430c7c

Attached sample.
[helloworld-4.0.20.zip](https://github.com/accordproject/cicero/files/3725606/helloworld-4.0.20.zip)

```
bash-3.2$ ~/git/cicero/packages/cicero-cli/index.js parse 
13:23:12 - info: Using current directory as template folder
13:23:12 - info: Loading a default sample.md file.
13:23:13 - info:
{
  "$class": "org.accordproject.helloworld.HelloWorldClause",
  "clauseId": "1882331a-3ee7-4eb1-92ac-1f244b81d09e",
  "name": "Fred Blogs",
  "myConcept": {
    "$class": "org.accordproject.helloworld.MyConcept",
    "myDateTime": "2000-10-10"
  }
}
```
